src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -362,6 +377,7 @@ namespace OpenRCT2::Ui::Windows
             // Turn on view clipping when the window is opened.
             if (mainWindow != nullptr)
             {
+                gClipHeightSeeThrough = false;
```

Comment:
Why should we reset on open? I guess we could remember the last option, no? 
Also isn't this reduntand if it is already initialized to `false` and reset to `false` on `OnClose`?

Reply:
Good point. If the cut height is retained after closing the window, then this should also apply to the see-through option.

I've removed both `gClipHeightSeeThrough = false;`.